xla/service/gpu/custom_call_test.cc

```diff
@@ -611,6 +611,25 @@ TEST_F(CustomCallTest, WithCalledComputation) {
   EXPECT_THAT(result.data<float>(), ::testing::Each(42));
 }
 
+TEST_F(CustomCallTest, WithCalledComputationAndLayouts) {
+  auto shape = ShapeUtil::MakeShapeWithDenseLayout(F32, {128}, {0});
```

Comment:
Can you please adjust the test so that a layout constraint is actually meaningful? For rank 1 it doesn't matter.